### ISSUE TITLE
SSL support to SIM7600

### ISSUE THREAD

**Issue by ramonpaolo** [vshymanskyy/TinyGSM#786]
[x] I have read the Troubleshooting section of the ReadMe

## What type of issues is this?

[x] Request to support a new module
[ ] Bug or problem compiling the library
[ ] Bug or issue with library functionality (ie, sending data over TCP/IP)
[ ] Question or request for help

### What are you working with?

Modem: SIM 7600
Main processor board: Uno
TinyGSM version: 0.11.7
Code: 

The SIM 7600 not have yet a support to SSL connections!

**Comment by floBik** [comment]
There is already a pull request (#767 ) from @Matt-Stedman and myself to add this feature, awaiting approval. 
Perhaps @vshymanskyy or @SRGDamia1 could take a look at this pr.

**Comment by Gspohu** [comment]
Any news on it ?

**Comment by don41382** [comment]
2 month past again. Any update when this will be merged?

**Comment by Pakingster** [comment]
Any updates?

**Comment by floBik** [comment]
I recently created a new PR(#808 ) with a bump to the current TinyGSM version and some refactoring.

**Comment by Pakingster** [comment]
> I recently created a new PR(#808 ) with a bump to the current TinyGSM version and some refactoring.

Is MQTT TLS connection supported, what a chance you have any example for this? 
Thanks

**Comment by rmallah** [comment]
this is needed, @vshymanskyy  could you pls review the PR.

**Comment by Matt-Stedman** [comment]
> > I recently created a new PR([#808](https://github.com/vshymanskyy/TinyGSM/pull/808) ) with a bump to the current TinyGSM version and some refactoring.
> 
> Is MQTT TLS connection supported, what a chance you have any example for this? Thanks

Yes, it is supported - I'll see about making an example shortly. (I've tested with AWS IoT Core).

However, seriously please @vshymanskyy this has been open and ready for nearly 2 years, @floBik and I worked hard on it, I believe it's ready to merge, and the community really need these features - if you have feedback please let us know otherwise I don't understand what the hold up is?

**Comment by apeman76** [comment]
bump @vshymanskyy

**Comment by alex-reusables** [comment]
adding that we'd love to see this issue resolved as well @vshymanskyy
